Debian-specific Makefile adjustments
Patch author: Ryan Finnie <ryan@finnie.org>

diff -ruN digitemp-3.5.0ds1.orig/Makefile digitemp-3.5.0ds1/Makefile
--- digitemp-3.5.0ds1.orig/Makefile	2007-01-01 10:47:02.000000000 -0800
+++ digitemp-3.5.0ds1/Makefile	2012-04-16 09:42:12.000000000 -0700
@@ -11,10 +11,13 @@
 
 VERSION = 3.5.0
 
+SRCDIR = .
+VPATH = $(SRCDIR)
 
-CC	= gcc
-CFLAGS	= -I./src -I./userial -O2 # -g -Wall
-LIBS	= -lm
+#CC	= gcc
+CFLAGS	= -I$(SRCDIR)/src -I$(SRCDIR)/userial
+CFLAGS += $(DEBCFLAGS)
+#LIBS	= -lm
 
 OBJS		=	src/digitemp.o src/device_name.o
 HDRS		= 	src/digitemp.h src/device_name.h
@@ -134,6 +137,9 @@
 		rm -f core *.asc 
 		rm -f perl/*~ rrdb/*~ .digitemprc digitemp-$(VERSION)-1.spec
 		rm -rf digitemp-$(VERSION)
+		rm -f digitemp_DS9097
+		rm -f digitemp_DS9097U
+		rm -f digitemp_DS2490
 
 # Sign the binaries using gpg (www.gnupg.org)
 # My key is available from www.brianlane.com
